aptos-move/e2e-move-tests/src/tests/scripts.rs

```diff
@@ -5,10 +5,118 @@ use crate::{assert_success, build_package, tests::common, MoveHarness};
 use aptos_language_e2e_tests::account::TransactionBuilder;
 use aptos_types::{
     account_address::AccountAddress,
+    on_chain_config::FeatureFlag,
     transaction::{Script, TransactionArgument},
 };
 use move_core_types::language_storage::TypeTag;
 
+#[test]
+fn test_script_with_object_parameter() {
+    let mut h = MoveHarness::new();
+
+    let alice = h.new_account_at(AccountAddress::from_hex_literal("0xcafe").unwrap());
+    let bob = h.new_account_at(AccountAddress::from_hex_literal("0xface").unwrap());
+    let root = h.aptos_framework_account();
+
+    let mut build_options = aptos_framework::BuildOptions::default();
+    build_options
+        .named_addresses
+        .insert("example_addr".to_string(), *alice.address());
+
+    let result = h.publish_package_with_options(
+        &alice,
+        &common::test_dir_path("../../../move-examples/fungible_asset/managed_fungible_asset"),
+        build_options.clone(),
+    );
+
+    assert_success!(result);
+    let result = h.publish_package_with_options(
+        &alice,
+        &common::test_dir_path("../../../move-examples/fungible_asset/managed_fungible_token"),
+        build_options.clone(),
+    );
+    assert_success!(result);
```

Comment:
Seems like we publish twice, one time is enough?

Reply:
I think one is asset and the other is a token.

Reply:
I am blind, thanks!